models/intermediate/int_zendesk__schedule_spine.sql

```diff
@@ -36,17 +36,18 @@ with timezone as (
     from timezone 
     left join daylight_time 
         on timezone.time_zone = daylight_time.time_zone
+        and timezone.source_relation = daylight_time.source_relation
```

Comment:
I am now running into this issue when running this model on the `zendesk_yashwanth` schema. Do you have any understanding of why this would be ocurring?

![image](https://github.com/fivetran/dbt_zendesk/assets/74217849/68a7cf49-e22d-42b2-8e88-8ab79d212047)

Reply:
hmmmm oddly enough i am not getting this error ...